modules/spring-config/src/main/java/org/mule/config/spring/parsers/processors/CheckRequiredAttributesWhenNoChildren.java

```diff
@@ -17,17 +17,19 @@
 public class CheckRequiredAttributesWhenNoChildren extends CheckRequiredAttributes
 {
     private String elementName;
+    private String elementNamespaceUrl;
 
-    public CheckRequiredAttributesWhenNoChildren(String[][] attributeNames, String elementName)
+    public CheckRequiredAttributesWhenNoChildren(String[][] attributeNames, String elementName, String elementNamespaceUrl)
```

Comment:
it is not ideal to force all uses of this constructor to change for a functionality you only need just now (presumably in just one place). Either add a constructor which preserves the old behaviour or create a new subclass of CheckRequiredAttributesWhenNoChildren which is namespace aware. I'm open to other solutions as long a they don't have an impact as big as this one